index.js

```diff
@@ -35,13 +41,13 @@ export const AutocompleteInput = (props) => {
   onShowResults && onShowResults(showResults);
   return (
     <View style={[styles.container, containerStyle]}>
-      <View style={[styles.inputContainer, inputContainerStyle]}>{renderTextInput(props)}</View>
```

Comment:
Removed `props` here because it wasn't actually being used by the `renderTextInput` function. That function is pulling props directly from the component.